{{ cookiecutter.package_name|slugify }}/.gitignore

```diff
@@ -58,3 +58,6 @@ __pycache__/
 
 # VS Code
 .vscode/
+
+# Docs
+docs
```

Comment:
Yes, I was thinking of avoiding all the html docs in the repo, since html is a bit ugly to track in git and it's not necessarily up to speed with the project if one forgets to build the docs before pushing and  these docs are generated from the code (docstrings, ...) which is the single source of truth one would want to maintain.
But I understand it's comfortable for external parties to view the docs directly from gitlab or github.
I can remove that change for now.
Would a separate PR enforcing the rebuild of the docs in the last pre-commit step make sense for you?

Reply:
Would remove this change for now indeed. As for a PR that builds the docs: I'm actually not fully satisfied yet with `pdoc`, perhaps we should standardise on another tool -- but let's discuss that offline.

Reply:
oki